tests/validation/cattlevalidationtest/core/test_github.py

```diff
@@ -0,0 +1,75 @@
+import urlparse
+from common_fixtures import *  # NOQA
+from selenium import webdriver
+import json
+
+# test the github auth workflow
+
+
+if_github = pytest.mark.skipif("os.environ.get('API_AUTH_GITHUB"
+                               "_TEST_USER') == 'false'",
+                               reason='API_AUTH_GITHUB_TEST_USER is not set')
```

Comment:
This line is broken... this works...
if_github = pytest.mark.skipif(os.environ.get("API_AUTH_GITHUB_TEST_USER") is None, reason='API_AUTH_GITHUB_TEST_USER is not set')